### PR TITLE
Pagos20

### PR DESCRIPTION
**Pull request by EmmanuelJCS**
Se agrega el complemento de pagos

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@
 /node_modules
 /package-lock.json
 
+/nbproject/private/
```
Comment (eclipxe13) [inline .gitignore:7]: Please, put this ignore paths on your environment git config file, it does not belong to the project itself.

File: development/ElementsMaker/specifications/pagos20.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    "root-element": "Pagos",
+    "structure": {
+        "Totales": {},
+        "Pago": {
+            "multiple": true,
+            "DoctoRelacionado": {
+                "multiple": true,
+                "ImpuestosDR": {
+                    "RetencionesDR": {
+                        "multiple": true,
```
Comment (eclipxe13) [inline development/ElementsMaker/specifications/pagos20.json:17]: RetencionesDR should not be multiple

File: development/ElementsMaker/specifications/pagos20.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                            "multiple": true
+                        }
+                    },
+                    "TrasladosDR": {
+                        "TrasladoDR": {
+                            "multiple": true
+                        }
+                    }
+                },
+                "TrasladosDR": {
```
Comment (eclipxe13) [inline development/ElementsMaker/specifications/pagos20.json:28]: I think TrasladosDR is repeated

File: tests/CfdiUtilsTests/Elements/Pagos20/Pagos20Test.php [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+final class Pagos20Test extends ElementTestCase
+{
+
+    public function testPagos20(): void
+    {
+        $element = new Pagos();
+        $this->assertElementHasName($element, 'pagos20:Pagos');
+        $this->assertElementHasFixedAttributes($element, [
+            'xmlns:cartaporte20' => 'http://www.sat.gob.mx/Pagos20',
```
Comment (eclipxe13) [inline tests/CfdiUtilsTests/Elements/Pagos20/Pagos20Test.php:29]: The XML prefix should be xmlns:pagos20

File: nbproject/project.properties [hunk 5]
```diff
@@ -0,0 +1,7 @@
+include.path=${php.global.include.path}
```
Comment (eclipxe13) [inline nbproject/project.properties:1]: This file should not be included in the project

File: nbproject/project.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (eclipxe13) [inline nbproject/project.xml:1]: This file should not be included in the project

**Review by eclipxe13 — CHANGES_REQUESTED** [review]
I think files on namespace CfdiUtils\Elements\Pagos20 are missing from this PR, please, include them

File: .gitignore (hunk 1 above)
Comment (EmmanuelJCS) [reply .gitignore:7]: Solved

File: development/ElementsMaker/specifications/pagos20.json (hunk 2 above)
Comment (EmmanuelJCS) [reply development/ElementsMaker/specifications/pagos20.json]: Solved

File: tests/CfdiUtilsTests/Elements/Pagos20/Pagos20Test.php (hunk 4 above)
Comment (EmmanuelJCS) [reply tests/CfdiUtilsTests/Elements/Pagos20/Pagos20Test.php]: Solved

File: development/ElementsMaker/specifications/pagos20.json (hunk 3 above)
Comment (EmmanuelJCS) [reply development/ElementsMaker/specifications/pagos20.json]: Solved

File: nbproject/project.properties (hunk 5 above)
Comment (EmmanuelJCS) [reply nbproject/project.properties:1]: Solved

File: nbproject/project.xml (hunk 6 above)
Comment (EmmanuelJCS) [reply nbproject/project.xml:1]: Solved

**Pull request closed without merging**